### ISSUE TITLE
Black formatter defaults to writing to disk instead of using available stdio

### ISSUE THREAD

**Issue by smackesey** [creativenull/diagnosticls-configs-nvim#27]
Here's this repo's config for `black`:

```lua
local fs = require('diagnosticls-configs.fs')

return {
  sourceName = 'black',
  command = fs.executable('black'),
  args = { '%filepath' },
  doesWriteToFile = true,
  rootPatterns = {
    '.git',
    'pyproject.toml',
    'setup.py',
  },
```

This command writes to disk (`doesWriteToFile = true`). Isn't it more natural for a language server formatter to use stdio? This is also the default behavior for formatters in diagnosticls. Does this repo have a differetn philosophy about default behavior for formatters? To change black to use stdio by default, one could use this:

```lua
local fs = require('diagnosticls-configs.fs')

return {
  sourceName = 'black',
  command = fs.executable('black'),
  args = { '-' },
  rootPatterns = {
    '.git',
    'pyproject.toml',
    'setup.py',
  },
```

**Comment by creativenull** [comment]
Some formatters do not use stdout to output the formatted code (like eslint), some of them only print out the diff to stdout. So I usually just have them write to the file. So no philosophy, in general that I follow for this.

But in this case I don't remember why I went with that 🤔 and you're right `black` formatter can print output to stdout.

I try to follow what is listed in the [diagnostic languageserver formatters wiki](https://github.com/iamcco/diagnostic-languageserver/wiki/Formatters), I guess this time I didn't.

I'll make the changes

**Comment by creativenull** [comment]
Should be fixed in a0ebf06c
